mypy.ini

```diff
@@ -0,0 +1,7 @@
+[mypy]
```

Comment:
I _think_ this config can go in pyproject.toml? If it's possible can you put it in there?